hooks/post_gen_project.py

```diff
@@ -9,6 +9,7 @@
 
 
 def __gen_init__(module_name):
+    """Generate __init__.py file for namespace module."""
```

Comment:
just adding a general a single-line docstring for functions in `post_gen_project.py`